subsys/mgmt/mcumgr/Kconfig

```
@@ -315,51 +315,67 @@ config MCUMGR_SMP_BT_AUTHEN
 	  Enables encrypted and authenticated connection requirement to
 	  Bluetooth SMP transport.
 
-config MCUMGR_SMP_BT_LATENCY_CONTROL
-	bool "Request low latency connection when handling SMP commands"
+config MCUMGR_SMP_BT_CONN_PARAM_CONTROL
+	bool "Request specific connection parameters for SMP packet exchange"
 	depends on SYSTEM_WORKQUEUE_PRIORITY < 0
+	depends on BT_GAP_PERIPHERAL_PREF_PARAMS
 	help
-	  Enables support for requesting low latency connection parameter when
+	  Enables support for requesting specific connection parameters when
 	  SMP commands are handled. This option allows to speed up the command
 	  exchange process.
 	  Its recommended to enable this if SMP is used for DFU.
 
-config MCUMGR_SMP_BT_LATENCY_CONTROL_DEFAULT_LATENCY
-	int "Default value for connection latency"
-	depends on MCUMGR_SMP_BT_LATENCY_CONTROL
-	default BT_PERIPHERAL_PREF_LATENCY if BT_GAP_PERIPHERAL_PREF_PARAMS
-	default 99
+if MCUMGR_SMP_BT_CONN_PARAM_CONTROL
+config MCUMGR_SMP_BT_CONN_PARAM_CONTROL_MIN_INT
+	int "Minimum connection interval for SMP packet exchange"
+	default 6
+	range 6 65535
+	help
+	  Minimum connection interval in 1.25ms units used during the exchange of SMP packets.
+
+config MCUMGR_SMP_BT_CONN_PARAM_CONTROL_MAX_INT
+	int "Maximum connection interval for SMP packet exchange"
+	default 9
+	range 6 65535
```

Range of 0x0006 to 0x0C80. 7.5ms to 4 seconds.

```suggestion
	range 6 3200
```